### DISCUSSION TITLE
getting "Call to a member function authorise() on null" when going to "Overview" > "Extension Updates"

### DISCUSSION THREAD

**Discussion by SumCompanyInc** [akeeba/panopticon#277] [Q&A (Community Support)] [answered]
i dont know if anyone else has experienced this issue, i dont know if it worked before, but when i go to the menu, Overview > Extension Updates, I ran into an issue crash: 

Call to a member function authorise() on null
[View template: ROOT/tmp/compiled_templates/51cff6adc6e5c4eb1bd0fee7ad9fa33f250aee6c.php line 331, compiled from ROOT/ViewTemplates/Extupdates/default.blade.php]

https://snipboard.io/ul163F.jpg

**Comment by nikosdion** [comment]
Yes, this is already reported and resolved.

**Reply by SumCompanyInc** [reply]
@nikosdion really appreciate all your efforts and your contributions to joomla community. sorry if i missed the point, when you say resolved.. a fix will be released in a future version or it's already resolved in v1.0.3? im currently using v1.0.3, and the latest is v1.0.3 -  and it's displaying error for me. i just don't know what i should be expecting when i click into "extension updates" - i am thinking i should be holding out for a future update to see a fix.. ?

**Reply by brianteeman** [reply, marked as answer]
It has been fixed and you will get the fix with the next release which will be 1.0.4

**Reply by SumCompanyInc** [reply]
@brianteeman thank you also! y'all are the best!
